java/src/main/resources/ui/root_list.js

```diff
@@ -23,7 +23,7 @@ function initRootList() {
   setButter('Loading root jobs...');
   $.ajax({
     type: 'GET',
-    url: 'rpc/list' + window.location.search,
+    url: 'rpc/list' + _.escape(window.location.search),
```

Comment:
At least, there is a warning on the review too. Not sure if it will be gone after these changes or are the same we had before

![image](https://github.com/Worklytics/appengine-pipelines/assets/75276364/37bcda3e-754b-4b43-ad7b-41c9977dedfe)
![image](https://github.com/Worklytics/appengine-pipelines/assets/75276364/37bcda3e-754b-4b43-ad7b-41c9977dedfe)

Reply:
fixed now